webrtc-stats.html

```diff
@@ -1136,8 +1136,8 @@
               </dt>
               <dd>
                 <p>
-                  Only valid for video MediaStreamTracks and represents the width of the video
-                  frame for this track.
+                  Only valid for video MediaStreamTracks and represents the width of the last
+                  decoded video frame for this track.
```

Comment:
I think this should be "last processed video frame" - for locally sourced tracks, there may not be any decoding involved (at least for RGB cameras).

Should we also say that before the first frame is processed, this attribute is missing?